### PR TITLE
task NXP 32954 k8s app user guide

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,342 @@
...
+
+Then, access the Nuxeo server with [http://localhost:8080/](http://localhost:8080/).
+
+## Scaling
+
+See the [Architecture](https://github.com/nuxeo/nuxeo-helm-chart/?tab=readme-ov-file#architecture) section of the Nuxeo Helm chart documentation.
+
+## Back Up and Restore
+
+See the Nuxeo documentation abount [Backup and Restore](https://doc.nuxeo.com/nxdoc/backup-and-restore/).
```
Comment (troger) [inline README.md]:
```suggestion
See the Nuxeo documentation about [Backup and Restore](https://doc.nuxeo.com/nxdoc/backup-and-restore/).
```

File: README.md [hunk 2]
```diff
@@ -1 +1,342 @@
...
+
+## Upgrade the application
+
+### Prepare the environment
+
+We recommend to backing up your data before starting the upgrade.
```
Comment (troger) [inline README.md]:
```suggestion
We recommend backing up your data before starting the upgrade.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,342 @@
...
+gcloud container clusters delete "$CLUSTER" --zone "$ZONE"
+```
+
+## License
+
+View [license information](https://doc.nuxeo.com/nxdoc/licenses/) for the software contained in the Nuxeo container image.
+
+As with all container images, these likely also contain other software which may be under other licenses (such as Bash, etc from the base distribution, along with any direct or indirect dependencies of the primary software being contained).
+
+As for any pre-built image usage, it is the image user's responsibility to ensure that any use of this image complies with any relevant licenses for all software contained within.
```
Comment (troger) [inline README.md]:
```suggestion
As with any pre-built image, it is the user's responsibility to ensure compliance with all relevant licenses for the software included in the image.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,342 @@
...
+export PUBLIC_SERVICE_AND_INGRESS_ENABLED=true
+```
+
+> [!WARNING] Warning about GKE Ingress
+
+When using the built-in Ingress controller provided by Google Kubernetes Engine, [GKE Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress), the `X-Forwarded-Port` request header is not set. Thus, you need to configure it as a [custom request header](https://cloud.google.com/load-balancing/docs/https/custom-headers#working-with-request).
+
+Alternately, you can:
+- Use another Ingress controller such as [NGINX Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/).
+- Configure  [Cloud DNS for GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-dns), in which case you need to set the `virtualHost` parameter of the `nuxeo` subchart with the DNS hostname, see [Install the Helm chart](#install-the-helm-chart).
```
Comment (kevinleturc) [inline README.md]: For information:
- https://hyland.atlassian.net/browse/NXP-32992
- https://github.com/nuxeo/nuxeo-lts/pull/2632 (I'm doing a bit of rework)

**Review by kevinleturc — APPROVED** [review]
Good 👌

File: README.md (hunk 4 above)
Comment (ataillefer) [reply README.md]: We can always remove this warning once the improvement is merged!

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**